### ISSUE TITLE
Raw ampersand in Onyx10.12.install.recipe is throwing errors when your repo is added

### ISSUE THREAD

**Issue by scriptingosx** [autopkg/tallfunnyjew-recipes#11]
Line 6 of that file has a raw ampersand, needs to be escaped.

```
WARNING: plist error for /Users/armin/Library/AutoPkg/RecipeRepos/com.github.autopkg.tallfunnyjew-recipes/Onyx10.12/Onyx10.12.install.recipe: Error Domain=NSCocoaErrorDomain Code=3840 "Encountered unknown ampersand-escape sequence at line 6" UserInfo={NSDebugDescription=Encountered unknown ampersand-escape sequence at line 6, kCFPropertyListOldStyleParsingError=Error Domain=NSCocoaErrorDomain Code=3840 "Malformed data byte group at line 1; invalid hex" UserInfo={NSDebugDescription=Malformed data byte group at line 1; invalid hex}}
```

```
$ plutil -lint /Users/armin/Library/AutoPkg/RecipeRepos/com.github.autopkg.tallfunnyjew-recipes/Onyx10.12/Onyx10.12.install.recipe
/Users/armin/Library/AutoPkg/RecipeRepos/com.github.autopkg.tallfunnyjew-recipes/Onyx10.12/Onyx10.12.install.recipe: Encountered unknown ampersand-escape sequence at line 6
```

**Comment by homebysix** [comment]
This will be resolved when PR #7 is merged.

**Comment by tallfunnyjew** [comment]
PR #7 was merged. closing and thanks for the help, fellers.
